CONTRIBUTING.rst

```diff
@@ -0,0 +1,70 @@
+Contributing and Maintenance Guide
+==================================
+
+Deploying bedevere
+------------------
+
+Bedevere web service is deployed to Heroku, which is managed by The PSF.
+
+Changes are supposed to be deployed to Heroku immediately after the PR gets merged,
+however, due to recent `security incident`_,
+the Heroku GitHub integration is broken.
+
+Until this gets resolved, maintainers have to deploy bedevere to Heroku manually.
+
+Who can deploy to Heroku
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+People listed as Collaborator/Member on the ``bedevere`` Heroku app can deploy to Heroku.
+Additionally, Heroku admins can also do it.
+
+Collaborators/members:
+
+- @Mariatta
+- @ambv
+- @brettcannon
+
+How to deploy manually to Heroku
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Install Heroku CLI
+
+   Details at: https://devcenter.heroku.com/articles/heroku-cli
+  
+#. Login to Heroku CLI on the command line and follow instructions
+
+   ::
```

Comment:
```suggestion
#. Login to Heroku CLI on the command line and follow instructions::
```

You can save a couple of lines this way (it will also show a `:` at the end of the sentence. (same below)